sdk/evaluation/azure-ai-evaluation/CHANGELOG.md

```diff
@@ -5,9 +5,23 @@
 ### Features Added
 
 - Added `type` field to `AzureOpenAIModelConfiguration` and `OpenAIModelConfiguration`
+- The following evaluators now support `conversation` as an alternative input to their usual single-turn inputs:
+  - `ViolenceEvaluator`
+  - `SexualEvaluator`
+  - `SelfHarmEvaluator`
+  - `HateUnfairnessEvaluator`
+  - `ProtectedMaterialEvaluator`
+  - `IndirectAttackEvaluator`
+  - `CoherenceEvaluator`
+  - `RelevanceEvaluator`
+  - `FluencyEvaluator`
+  - `GroundednessEvaluator`
```

Comment:
Can similarity inhert from prompty base class?

Reply:
Eventually maybe, but not now. It's not on the list of evals that need to support conversations because it's a ground truth-based evaluator.